app/views/homerooms/show.html.erb

```diff
@@ -74,19 +74,22 @@
             <th class="risk sort-default" data-sort-method="reverse_number">
               <span class="table-header">Risk</span>
             </th>
-            <th class="program">
+            <!-- The data-sort-method attribute forces Tablesort to use a specific sorting method, instead of the default sort it uses. You can see the method at: app/assets/javascripts/roster/sorts/prog_assign_sort.js -->
```

Comment:
Nice, thanks for adding this in!  Using HTML comments will send this text all the way down to end-users in the browser though, so comments in ERB files are better expressed like this:

```
<%
 # The data-sort-method attribute forces Tablesort to use a specific sorting method, instead of the default sort it uses.
 # You can see the method at: app/assets/javascripts/roster/sorts/prog_assign_sort.js
%>
```

So that it's only visible to us developers. :)